package.json

```diff
@@ -63,6 +64,7 @@
     "eslint-plugin-jsx-a11y": "^6.0.3",
     "eslint-plugin-react": "^7.9.1",
     "get-graphql-schema": "^2.1.1",
```

Comment:
Should we delete graphql-get-schema if we've got the graphql cli in now?

Reply:
Opps, I didn't see it. I'm not sure which one is better.